l10n_de_tax_statement/models/account_move.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 BIG-Consulting GmbH(<http://www.openbig.org>)
+# Copyright 2019-2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    l10n_de_tax_statement_id = fields.Many2one(
+        "l10n.de.tax.statement", "Statement", copy=False
```

Comment:
To avoid the following warning:

`2021-08-25 08:45:20,176 14141 WARNING l10n_de_tax_statement_14_1 odoo.addons.base.models.ir_model: Two fields (l10n_de_tax_statement_id, statement_id) of account.bank.statement.line() have the same label: Statement. `

```suggestion
        "l10n.de.tax.statement", copy=False
```